Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2227,6 +2227,39 @@ extension BrowserViewController: WKUIDelegate {
             self.tabManager.removeTab(tab)
         }
     }
+    
+    func webView(_ webView: WKWebView, shouldPreviewElement elementInfo: WKPreviewElementInfo) -> Bool {
+        guard let url = elementInfo.linkURL else { return false }
+        return !Storage.isIgnoredURL(url)
+    }
+    
+    func webView(_ webView: WKWebView, commitPreviewingViewController previewingViewController: UIViewController) {
+        guard let previewViewController = previewingViewController as? PreviewViewController else { return }
+        tabManager.selectedTab?.webView?.load(URLRequest(url: previewViewController.url))
+    }
+    
+    func webView(_ webView: WKWebView, previewingViewControllerForElement elementInfo: WKPreviewElementInfo, defaultActions previewActions: [WKPreviewActionItem]) -> UIViewController? {
```

Comment:
Split this line, it's a bit too long, we usually try to not exceed 130 characters